src/components/views/project/NotAvailableProject.tsx

```diff
@@ -0,0 +1,150 @@
+import Image from 'next/image';
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { client } from '@/apollo/apolloClient';
+import { IProject } from '@/apollo/types/types';
+import { FETCH_HOME_PROJECTS } from '@/apollo/gql/gqlProjects';
+import { EDirection, gqlEnums } from '@/apollo/types/gqlEnums';
+import { brandColors, neutralColors, H4, H6 } from '@giveth/ui-design-system';
+import { mediaQueries } from '@/lib/constants/constants';
+import ProjectCard from '@/components/project-card/ProjectCard';
+
+const projectsToFetch = 12;
+const projectsToShow = 3;
+const NotAvailableProject = () => {
+	const [suggestedProjects, setSuggestedProjects] = useState<IProject[]>([]);
+	const [listPosition, setListPosition] = useState<number>(0);
+
+	const moveList = (pos: number) => {
+		if (suggestedProjects.length === 0) return;
+		const newPos = listPosition + pos;
+		if (newPos >= 0 && newPos < projectsToFetch / projectsToShow) {
+			setListPosition(newPos);
+		}
+	};
+
+	useEffect(() => {
+		const fetchSuggestedProjects = async () => {
+			const variables: any = {
+				limit: projectsToFetch,
+				orderBy: {
+					field: gqlEnums.QUALITYSCORE,
+					direction: EDirection.DESC,
+				},
+			};
+			const { data } = await client.query({
+				query: FETCH_HOME_PROJECTS,
+				variables,
+				fetchPolicy: 'network-only',
+			});
+			setSuggestedProjects(data.projects?.projects);
+		};
+
+		fetchSuggestedProjects();
+	}, []);
+
+	return (
+		<Wrapper>
+			<Image
+				src='/images/missing-project.svg'
+				width={122.69}
+				height={112}
+			/>
+			<TitleText>Oops! This project is no longer available!</TitleText>
+			<SubtitleText>Take a look at similar projects</SubtitleText>
+			<Slider>
+				{listPosition !== 0 ? (
+					<CaretLeft
+						src='/images/caret_right.svg'
+						onClick={() => moveList(-1)}
+					/>
```

Comment:
we have it on the design system, I suggest using it (CaretRight, CaretLeft, and ...).

Reply:
The design is different than the ones on the design system :c